### PR TITLE
blog describing an integration of h-alerts with opentracing

### PR DESCRIPTION
**Pull request by jmazzitelli**

### CODE REVIEW

File: src/main/jbake/content/blog/2017/09/alerts-opentracing.adoc [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+```
+minikube start --cpus 4 --memory 8192
+
+minikube dashboard
+```
+
+=== DEPLOY PROMETHEUS
+
+```
+kubectl create -f https://raw.githubusercontent.com/coreos/prometheus-operator/master/bundle.yaml
```
Comment (objectiser) [inline src/main/jbake/content/blog/2017/09/alerts-opentracing.adoc]: Yesterday I had to update the example instructions to base it on prom-operation v0.11.0, to get it working consistently on kubernetes and openshift, so may be best if you update this to be:
`kubectl create -f https://raw.githubusercontent.com/coreos/prometheus-operator/v0.11.0/bundle.yaml`.

I will need to update the other blog posts to make sure they are consistent. So this now just uses the v1alpha1.

**Review by objectiser — APPROVED** [review]
Very good - only one minor change to align with recent update to the example.

File: src/main/jbake/content/blog/2017/09/alerts-opentracing.adoc (hunk 1 above)
Comment (jmazzitelli) [reply src/main/jbake/content/blog/2017/09/alerts-opentracing.adoc]: see latest commit that makes this change.

Comment (objectiser) [reply src/main/jbake/content/blog/2017/09/alerts-opentracing.adoc]: Thanks 👍

**Pull request merged**